source/lib-intro.tex

```diff
@@ -900,14 +900,19 @@
 \end{footnote}
 
 \pnum
-Whenever a name \tcode{x} defined in the standard library is mentioned,
-the name \tcode{x} is assumed to be fully qualified as
-\tcode{::std::x},
-unless explicitly described otherwise. For example, if the \effects element
-for library function \tcode{F} is described as calling library function \tcode{G},
-the function
-\tcode{::std::G}
-is meant.
+Unless otherwise specified,
+argument-dependent name lookup\iref{basic.lookup.argdep} is only performed
+within the standard library
+\begin{itemize}
+\item
+for \grammarterm{operator-function-id}s and
+\item
+for the \grammarterm{identifier} \tcode{swap},
+in an overload-resolution context for swappable values\iref{swappable.requirements}.
+\end{itemize}
+\begin{note}
+An implementation could use \grammarterm{qualified-id}{s} to achieve that.
```

Comment:
I think it's a bit unclear what "that" refers to here. Maybe "Using a _qualified-id_ avoids argument-dependent lookup." instead?